components/ListView/BlueprintListView.js

```diff
@@ -80,14 +75,17 @@ BlueprintListView.propTypes = {
   handleShowModalDelete: PropTypes.func,
   blueprints: PropTypes.arrayOf(PropTypes.object),
   handleShowModalExport: PropTypes.func,
-  handleShowModalCreateImage: PropTypes.func
+  imageTypes: PropTypes.arrayOf(PropTypes.object).isRequired,
+  layout: PropTypes.shape({
+    setNotifications: PropTypes.func
+  })
```

Comment:
Curious: why is `setNotifications` in a separate `layout` object?

The `isRequired` is added in the first commit and removed in the second.

Reply:
Oh, this is a method that is implemented in the Layout component so basically whenever we want this pop up notification we need a reference to that component to call the setNotifications function. It's not good, but that's how it was before. 
I was thinking already how to make it better.

Reposting to fix this double change of the isRequired field.